pkg/convert/xliff2.go

```diff
@@ -145,6 +140,111 @@ func ToXliff2(translation model.Translation) ([]byte, error) {
 
 // helpers
 
+// messageFromUnit converts Xliff2.0 unit element to model.Message.
+func messageFromUnit(u unit, original bool) (*model.Message, error) {
+	var (
+		err     error
+		decoder *xml.Decoder
+	)
+
+	m := &model.Message{
+		ID:          u.ID, // TODO: translation.messages[n].ID should be based on source text.
```

Comment:
Same,redundant TODO

Reply:
If you want to preserve original text, I can suggest adding local variable, containing that text.

e.g.
```go
if !original{
  message.Local($sourceText, mf2.Literal(...))
}
```

Reply:
Do we have to keep original text though?

Reply:
we lose the source tag when we convert back nonoriginal translation -> xliff.
However, I am unsure if it is important for apps, which utilize Xliff...